entrypoint/createcluster/createcluster_suite_test.go

```diff
@@ -7,23 +7,21 @@ import (
 
 	"github.com/rancher/distros-test-framework/factory"
 	"github.com/rancher/distros-test-framework/pkg/customflag"
-	"github.com/rancher/distros-test-framework/shared"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+var cluster *factory.Cluster
+
 func TestMain(m *testing.M) {
-	var err error
 	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
 	flag.Parse()
 
-	_, err = shared.EnvConfig()
-	if err != nil {
-		return
-	}
+	cluster = factory.ClusterConfig()
 
-	os.Exit(m.Run())
+	exitCode := m.Run()
```

Comment:
Any reason why we are adding a variable here and replacing the one liner?

Reply:
yeah its just a cleaner way to not tie this error directly , like first get whats happening then you exit
so if you need to deal with some extra test stuff, like log ,clean something or whatever you have a way to do so +

right now, makes no diff logically, just make more readable

Reply:
I was going to comment the same thing 😅 I prefer the one liner myself, but I'm fine either way here.

Reply:
well let me know....this PR its really getting stuck here

Reply:
here on the PRs, not on this comment i meant

Reply:
I also think we should keep the one-liner os.Exit(m.Run())

Reply:
sure, removing then